crates/database/src/settlement_call_data.rs

```diff
@@ -0,0 +1,53 @@
+use {crate::auction::AuctionId, sqlx::PgConnection};
+
+#[derive(Debug, Clone, PartialEq, sqlx::FromRow)]
+pub struct SettlementCallData {
+    pub auction_id: AuctionId,
+    pub call_data: Vec<u8>,
+    pub uninternalized_call_data: Vec<u8>,
```

Comment:
I think you should be able to use our [ByteArray](https://github.com/cowprotocol/services/pull/1051/files) here to get nicer debug printing of the calldata bytes.

Reply:
Probably no need to add it here also (also `database` crate doesn't have `derivative` dependency).